### PR TITLE
Add example on JWT token generation/validation

### PR DESCRIPTION
**Pull request by rchavik**

### CODE REVIEW

**Comment by ADmad** [comment]
The index page doesn't seem like the right place for this. It should under the "JWT" section on `docs/en/authenticators.rst`.

**Comment by rchavik** [comment]
It's under Quick start, right after standard login example. I think it's a good place.

I'll move it anyway

File: docs/en/index.rst [hunk 1]
```diff
@@ -170,6 +170,51 @@ setter method::
         }
     }
 
+JWT Token Generation
+====================
+
+To use JWT authentication, we need to generate keys::
```
Comment (ADmad) [inline docs/en/index.rst:176]:
```suggestion
By default the `JwtAuthenticator` uses `HS256` symmetric key algorithm and uses
the value of `Cake\Utility\Security::salt()` as encryption key.

For enhanced security one can instead use the `RS256` asymmetric key algorithm.
You can generate the required keys for that as follows::
```

File: docs/en/index.rst [hunk 2]
```diff
@@ -170,6 +170,51 @@ setter method::
...
+                'token' => JWT::encode($payload, $privateKey, 'RS256'),
+            ];
+        } else {
+            $this->response = $this->response->withStatus(401);
+        }
+        $this->set(compact('json'));
+        $this->viewBuilder()->setOption('serialize', 'json');
+    }
+    
+Note that this requires setting up your `Applicaion` class as follows::
```
Comment (ADmad) [inline docs/en/index.rst:206]:
```suggestion
Note that this requires setting up your `Application` class as follows::
```

**Comment by rchavik** [comment]
Obsoleted by #378

**Pull request closed without merging**